### PR TITLE
Fixed autofill unreadable fields and target table hidden on small sizes

### PR DESCRIPTION
**Pull request by diego-ge**
Fixed autofill unreadable parameters and preventing target table from getting hidden if tile size too small

https://user-images.githubusercontent.com/44274704/164801643-7e681eb7-f615-4b5a-b886-21b775085407.mp4

### CODE REVIEW

File: common/src/main/webapp/less/style.less [hunk 1]
```diff
@@ -2243,3 +2244,25 @@ th.sticky-header,
...
+.ui.input input:-webkit-autofill:focus,
+.ui.input input:-webkit-autofill:active,
+.ui.form input:-webkit-autofill,
+.ui.form input:-webkit-autofill:hover,
+.ui.form input:-webkit-autofill:focus,
+.ui.form input:-webkit-autofill:active,
+input:-webkit-autofill,
+input:-webkit-autofill:hover,
+input:-webkit-autofill:focus,
+input:-webkit-autofill:active
```
Comment (cquiroz) [inline common/src/main/webapp/less/style.less:2263]: Does this only afect safari?

**Review by cquiroz — APPROVED** [review]
LGTM

File: common/src/main/webapp/less/style.less (hunk 1 above)
Comment (diego-ge) [reply common/src/main/webapp/less/style.less:2263]: No, it shouldn't, actually the video was done using chrome.

Comment (cquiroz) [reply common/src/main/webapp/less/style.less:2263]: Is this available as a standard field. In theory we should be able to generate the browser specific suffixes via postcss

File: common/src/main/webapp/less/style.less [hunk 2]
```diff
@@ -2243,3 +2244,25 @@ th.sticky-header,
...
+{
+  box-shadow: 0 0 0 30px var(--form-input-background) inset !important;
+  -webkit-box-shadow: 0 0 0 30px var(--form-input-background) inset !important;
```
Comment (cquiroz) [inline common/src/main/webapp/less/style.less:2266]: Could we investigate https://github.com/postcss/autoprefixer to avoid these?

**Pull request merged**